### DISCUSSION TITLE
打开音频文件时 uosc常驻

### DISCUSSION THREAD

**Discussion by Qyyxzz** [hooke007/mpv_PlayKit#437] [mpv 通用]
打开音频文件的时候 uosc就会一直在 而视频图片文件不会 是设置方面的原因吗
![7a3aeabed830010fab2348c2f8cfdf69](https://github.com/hooke007/MPV_lazy/assets/137096681/4e35ca3d-e92e-4230-8960-77db39118928)
![9d566b66caed6e600216d50efe711a5b](https://github.com/hooke007/MPV_lazy/assets/137096681/3c60c324-e90b-4c6d-acb2-dadecd1c9847)
![80021659e7bd913fa1ca8a7fde965de8](https://github.com/hooke007/MPV_lazy/assets/137096681/c4bdf658-5e62-4a46-bb1a-09531751bd60)

**Comment by hooke007** [comment]
https://github.com/hooke007/MPV_lazy/blob/2a681161e227e2b56af3b3882149713e0513b6ef/portable_config/script-opts.conf#L469

**Reply by Qyyxzz** [reply]
感谢
